src/rtc.rs

```diff
@@ -158,17 +198,82 @@ impl Rtc {
         write_protection(&self.rtc, true);
     }
 
-    pub fn get_date(&self) -> Date {
-        let date;
+    pub fn get_config(&self) -> RtcConfig{
+        self.rtc_config
+    }
+
+    pub fn set_config(&mut self, apb1r1: &mut APB1R1, bdcr: &mut BDCR, pwrcr1: &mut pwr::CR1, clocks: Clocks, rtc_config : RtcConfig) {
```

Comment:
`clocks` is unused in the entire file? No reason to even pass it to the initial `Rtc` function..

Reply:
You are right.